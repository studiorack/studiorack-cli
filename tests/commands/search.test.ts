import { expect, test } from 'vitest';
import { cli } from '../shared';
import { RegistryType } from '@open-audio-stack/core';

test('Search packages', async () => {
  expect(cli(RegistryType.Plugins, 'search', 'XT')).toMatchSnapshot();
  expect(cli(RegistryType.Plugins, 'search', 'ZXT')).toMatchSnapshot();
});
