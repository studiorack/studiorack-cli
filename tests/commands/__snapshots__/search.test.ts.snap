// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Search packages 1`] = `
┌─────────────────────────┬─────────────┬─────────┬────────────┬─────────┬────────────────────────────────┐
│ Id                      │ Name        │ Version │ Date       │ License │ Tags                           │
├─────────────────────────┼─────────────┼─────────┼────────────┼─────────┼────────────────────────────────┤
│ tote-bag-labs/valentine │ Valentine   │ 1.0.1   │ 2024-04-27 │ gpl-3.0 │ Effect, Compressor, Distortion │
├─────────────────────────┼─────────────┼─────────┼────────────┼─────────┼────────────────────────────────┤
│ tesselode/flutterbird   │ Flutterbird │ 1.0.1   │ 2018-09-06 │ mit     │ Effect, Pitch, Modulation      │
├─────────────────────────┼─────────────┼─────────┼────────────┼─────────┼────────────────────────────────┤
│ surge-synthesizer/surge │ Surge XT    │ 1.3.4   │ 2024-08-11 │ gpl-3.0 │ Instrument, Synth, Modulation  │
└─────────────────────────┴─────────────┴─────────┴────────────┴─────────┴────────────────────────────────┘
`;

exports[`Search packages 2`] = `No results found`;
